src/api/features/features.api.js

```diff
@@ -529,108 +525,161 @@ export const identify = (config) => {
+        const { name = null, title = null, label = null } = featureMetadata.properties
+        const titleInCurrentLang = featureMetadata.properties[`title_${lang}`]
+        if (label) {
+            featureName = label
+        } else if (name) {
+            featureName = name
+        } else if (title) {
+            featureName = title
+        } else if (titleInCurrentLang) {
+            featureName = titleInCurrentLang
+        }
+    }
+
+    if (outputProjection.epsg !== LV95.epsg) {
+        if (featureExtent.length === 4) {
+            featureExtent = projExtent(LV95, outputProjection, featureExtent)
+        }
+    }
+
+    return new LayerFeature({
+        layer,
+        id: featureMetadata.id,
+        name: featureName,
+        data: featureHtmlPopup,
+        coordinates: getGeoJsonFeatureCoordinates(featureGeoJSONGeometry, LV95, outputProjection),
+        extent: featureExtent,
+        geometry: featureGeoJSONGeometry,
+    })
+}
+
 /**
  * Loads a feature metadata and tooltip content from this two endpoint of the backend
  *
- * - http://api3.geo.admin.ch/services/sdiservices.html#identify-features
+ * - https://api3.geo.admin.ch/services/sdiservices.html#feature-resource
  * - http://api3.geo.admin.ch/services/sdiservices.html#htmlpopup-resource
  *
  * @param {GeoAdminLayer} layer The layer from which the feature is part of
- * @param {String | Number} featureID The feature ID in the BGDI
+ * @param {String | Number} featureId The feature ID in the BGDI
  * @param {CoordinateSystem} outputProjection Projection in which the coordinates (and possible
  *   extent) of the features should be expressed
- * @param {String} lang The language for the HTML popup
+ * @param {String} lang The language for the HTML popup. Default is `en`.
  * @returns {Promise<LayerFeature>}
  */
-const getFeature = (layer, featureID, outputProjection, options = {}) => {
-    const { lang = 'en', coordinates = null, imageDisplay = null, mapExtent = null } = options
+const getFeature = (layer, featureId, outputProjection, lang = 'en') => {
     return new Promise((resolve, reject) => {
         if (!layer?.id) {
             reject('Needs a valid layer with an ID')
         }
-        if (!featureID) {
+        if (!featureId) {
             reject('Needs a valid feature ID')
         }
-        // combining the two requests in one promise
-        const topic = layer.getTopicForIdentifyAndTooltipRequests()
-        const featureUrl = `${API_BASE_URL}rest/services/${topic}/MapServer/${layer.id}/${featureID}`
-        const params = {
-            sr: LV95.epsgNumber,
-            lang: lang,
-        }
-        if (coordinates) {
-            params.coord = coordinates
-        }
-        if (imageDisplay) {
-            params.imageDisplay = imageDisplay
-        }
-        if (mapExtent) {
-            params.mapExtent = mapExtent
+        if (!outputProjection) {
+            reject('An output projection is required')
         }
```

Comment:
You will break some layer if you don't add those parameter to the GetFeature, see https://github.com/geoadmin/web-mapviewer/pull/933

Reply:
Without `coord`, `mapExtent` and `imageDisplay` the https://api3.geo.admin.ch/services/sdiservices.html#feature-resource don't return the correct data for certain layer that was the issue with the `Moorlandschaften AuLaV` layer so you need to pass them (maybe not all of them, just saw that the old viewer used all of them)

Reply:
that's weird, these params (especially `mapExtent` which seems the trigger to have the correct output) are not documented on this endpoint
https://api3.geo.admin.ch/services/sdiservices.html#htmlpopup-resource

Reply:
https://api3.geo.admin.ch/services/sdiservices.html#feature-resource doesn't mentioned them either

Reply:
I had a quick chat with Marcel, and the layer you mentioned is a bit of an edge case, as it has two different model to show the users. It will select the model depending on the resolution of the map, hence the need for all these params (especially `mapExtent` and `imageDisplay`)

I'll add them back to the HTML popup request with a bit of documentation